### PR TITLE
Use Bot's token instead of SSH Key

### PR DESCRIPTION
**Pull request by jderusse**
Use Bot's Personal Access Token instead of SSH Key (reduce number of secrets)

### CODE REVIEW

File: .github/workflows/website.yml [hunk 1]
```diff
@@ -16,19 +16,15 @@ jobs:
...
         uses: actions/checkout@v2
         with:
           path: 'source'
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
 
-      - name: Checkout Website repo via SSH
+      - name: Checkout Website repo
         run: |
-          git clone --branch master git@github.com:async-aws/async-aws.github.io.git output
+          git clone --branch master http://github.com/async-aws/async-aws.github.io output
```
Comment (Nyholm) [inline .github/workflows/website.yml]: Could you update the URL?

Comment (jderusse) [reply .github/workflows/website.yml]: thanks, fixed.

Comment (Nyholm) [reply .github/workflows/website.yml]: Could you rebase, it seams like there is a conflict?

**Comment by Nyholm** [comment]
This seamed to fail: 

https://github.com/async-aws/aws/runs/543246103?check_suite_focus=true#step:7:36

**Pull request merged**